services/user.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str, password: str, **kwargs) -> None:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if kwargs.get("email"):
+        user.email = kwargs["email"]
+
+    if kwargs.get("first_name"):
+        user.first_name = kwargs["first_name"]
+
+    if kwargs.get("last_name"):
+        user.last_name = kwargs["last_name"]
+
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(id=user_id)
+
+
+def update_user(user_id: int, **kwargs) -> None:
+    user = get_user_model().objects.get(id=user_id)
```

Comment:
why don't just use `get_user()` method which you previously created?

Reply:
Fixed all  ;)